### ISSUE TITLE
与keras

### ISSUE THREAD

**Issue by MQ99** [bojone/crf#7]
请问，keras搭建层结构，model.compile(loss=CRF.loss, optimizer='adam',metrics=[CRF.accuracy]）必须得有y_pred,应该怎么修改

**Comment by bojone** [comment]
不明白你想说什么？

**Comment by xiongyujia** [comment]
我也遇到相似的问题，就是在CRF的定义里loss和accuracy有两个参数y_true和y_pred，如果直接model.compile(loss=crf.loss, optimizer='adam',metrics=[crf.accuracy]，会报错
loss() missing 2 required positional arguments: 'y_true' and 'y_pred'
